charger/easee.go

```diff
@@ -292,9 +296,19 @@ func (c *Easee) ProductUpdate(i json.RawMessage) {
 	case easee.TOTAL_POWER:
 		c.currentPower = 1e3 * value.(float64)
 	case easee.SESSION_ENERGY:
+		//SESSION_ENERGY must not be set to 0 by Productupdates, they occur erratic
+		//Reset to 0 is done in case CHARGER_OP_MODE
+		if 0 == value.(float64) {
+			c.log.TRACE.Printf("ProductUpdate of SESSION_ENERGY to 0 ignored");
```

Comment:
We don't need this log, as we log all ProductUpdates above.
As value is never mutated that suffices to understand the flow from the logs (if we ever need to).
As a bonus, you can then also negate the if condition above, and get rid of the return below. Streamlines things.